app/src/main/res/values/strings.xml

```diff
@@ -180,11 +180,21 @@
     <!-- Turn On Sync Preferences -->
     <!-- Header of the Turn on Sync preference view -->
     <string name="preferences_sync">Turn on Sync</string>
+    <!-- Preference for pairing -->
+    <string name="preferences_sync_pair">Scan pairing code in desktop Firefox</string>
     <!-- Preference for account login -->
     <string name="preferences_sync_sign_in">Sign in</string>
     <!-- Preference for account creation -->
     <string name="preferences_sync_create_account">Create an account</string>
 
+    <!-- Pairing Feature strings -->
+    <!-- Instructions on how to access pairing -->
+    <string name="pair_instructions">Visit <b>firefox.com/pair</b> in Firefox on your computer to get your QR code.</string>
```

Comment:
TC is failing due to `    ERROR: Only plain text allowed, or one CDATA surrounded by whitespace at line 0, column 0 for pair_instructions`

UX wants this in bold, what should we do here?

Reply:
You might be able to try [this](https://stackoverflow.com/a/38569893)

Reply:
I had to use `fromHtml` though :(